lib/rucio/core/scope.py

```diff
@@ -64,12 +66,18 @@ def add_scope(scope, account, *, session: "Session"):
             raise Duplicate('Scope \'%s\' already exists!' % scope)
         else:
             raise RucioException(e)
-    except:
+    except Exception:
         raise RucioException(str(format_exc()))
 
 
 @read_session
-def bulk_add_scopes(scopes, account, skipExisting=False, *, session: "Session"):
+def bulk_add_scopes(
+    scopes: list[InternalScope],
```

Comment:
```suggestion
    scopes: Iterable[InternalScope],
```

`scopes` is only used as an iterable in this function